plugin.json

```diff
@@ -9,12 +9,13 @@
 		{ "hook": "static:app.load", "method": "init" },
 		{ "hook": "filter:admin.header.build", "method": "addMenuItem" },
 		{ "hook": "action:middleware.authenticate", "method": "authenticate"},
-		{ "hook": "action:settings.set", "method": "reloadSettings" }
+		{ "hook": "action:settings.set", "method": "reloadSettings" },
+		{ "hook": "filter:user.updateProfile", "method": "EnableUserFieldsToWrite" }
 	],
 	"less": [
 		"./less/style.less"
 	],
-	"scripts": [
+	"acpScripts": [
```

Comment:
this seems to be a thing in nodebb... I found some other guys doing the same in other plugins